src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int RANGE_BOUND = 100;
+    private ColorSupplier colorSupplier;
+    private Ball ball = new Ball();
```

Comment:
this should be a local variable - we create a whole new object each time this method is called, its scope should be local to the method. It's not the same as ColorSupplier - we don't need to recreate it, only use it